frontend/src/Editor/DataSourceManager/SourceComponents/Api/Graphql.schema.json

```diff
@@ -3,7 +3,22 @@
   "$id": "https://tooljet.io/Graphql.schema.json",
   "title": "Graphql datasource",
   "description": "A schema defining graphql datasource",
-  "type": "object",
+  "type": {
```

Comment:
CMIIW, according to JSON schema spec this [type](https://json-schema.org/understanding-json-schema/reference/type.html) would be invalid here, right?
Does it make sense to add this as the definition instead?

Reply:
Or we can use some other keyword which isn't part of the spec

Reply:
```
string
number
integer
object
array
boolean
null
```
Yeah true, only above values can exist inside type as per spec, I'll rename it to some other thing with a `$` in front to specify its custom